protocol/directory.go

```diff
@@ -102,3 +121,66 @@ func (d *ConiksDirectory) Register(req *RegistrationRequest) (
 		return NewRegistrationProof(ap, d.LatestSTR(), nil, Success)
 	}
 }
+
+func (d *ConiksDirectory) KeyLookup(req *KeyLookupRequest) (
+	*DirectoryProof, ErrorCode) {
+	ap, err := d.pad.Lookup(req.Username)
+	if err != nil {
+		return nil, ErrorDirectory
```

Comment:
> I think this should return a NewErrorResponse(), no?

You're right. It totally makes sense now. Thanks! Fixed in 5b780addc637e26562b30eab53b59673e473b3c8.